src/components/Widgets/Widget.tsx

```diff
@@ -49,18 +64,22 @@ export function Widget({ starterVariant }: WidgetProps) {
   const { multisigWidget, multisigSdkConfig } = getMultisigWidgetConfig();
   const { activeTab } = useActiveTabStore();
   const isGasVariant = activeTab === TabsMap.Refuel.index;
-
+  console.log('WIDGET', {
```

Comment:
Not sure it is expected to keep the console.log

Reply:
Thanks, it was still wip. So, I would have removed it - but well spotted! I have removed it by now.